05_outputs.Rmd

```diff
@@ -358,6 +358,167 @@ for (i in 1:length(col_names)) {
 
 ### change from label to xml
 
+The following code transform column headers, select_one and select_multiple values from Label to XML.
+
+Conditions required for the code to function correctly:
+1- The "label::English" column in KOBO Tool must contain unique values.
+2- The labeled data should be saved in the Excel file format.
+3- When selecting options for select_multiple questions, ensure that the options are separated by a forward slash ("/").
```

Comment:
So far the function is only taking "/" for the select multiple separators. In many contexts, "." is used as the separator for select_multiple questions. Can we make it dynamic in the function to take a parameter sm_separator maybe and by default is one of them, and user can choose. But than what, if the tool labels include "/" or ".".  😕

Reply:
Added sm_separator parameter.  Regarding if the tool labels contain more than "/" or "." --> it is a bit hard to cover all scenarios in one function. So the assumption is that we did not use "/" or "." when determining the "label English" column in the kobo tool.